src/framework/components/light/component.js

```diff
@@ -1119,6 +1119,51 @@ class LightComponent extends Component {
         return this.light.shadowUpdateOverrides;
     }
 
+    /**
+     * Sets the number of shadow samples used for soft shadows when the shadow type is
+     * {@link SHADOW_PCSS_32F}. This value must be a positive whole number starting at 1. Higher
+     * values result in smoother shadows but can significantly decrease performance. Defaults to 16.
```

Comment:
Doesn't state the upper bound mentioned in the PR description (128).

Reply:
There is no upper bound, the description only mentions what I exposed in the example (reasonable range).

Reply:
I fixed the description